### ISSUE TITLE
Determine and set a `main` landmark

### ISSUE THREAD

**Issue by adagar** [storybookjs/storybook#13794]
**Is your feature request related to a problem? Please describe**
When running Axe assertions, one of the errors is the lack of a main landmark. Landmarks are an essential tool for people using non-traditional modes of navigation, so this is a change that has high-impact for a11y-users and little to no impact for other users.

**Describe the solution you'd like**
An element, likely the DOM node that contains the visual and code view of elements should be changed from a generic `div` element to a `main` one.
![Screen Shot 2021-02-02 at 3 58 07 PM](https://user-images.githubusercontent.com/3698858/106661911-c5d86300-656f-11eb-9d42-531fb0306ef2.png)

NOTE: Due to the nature of nested documents (https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html#:~:text=The%20main%20landmark%20should%20be,should%20have%20a%20unique%20label.) a `<main>` landmark should also be determined for the content inside of the iFrame, though I understand that this content is considered user generated and beyond the scope of core story-book features.

**Are you able to assist to bring the feature to reality?**
Yes, I can. This should be a small change.

**Additional context**

**Comment by shilman** [comment]
Olé!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.2.0-alpha.26 containing PR #13827 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.
